content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -219,30 +201,35 @@ Batch billing
 
 Vendor bills can be processed and managed in batches in the *Accounting* app.
 
-To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
-:guilabel:`checkbox` at the top left of the page, beside the :guilabel:`Number` column, under the
-:guilabel:`Create` button. This selects all existing vendor bills with a :guilabel:`Posted` or
-:guilabel:`Draft` :guilabel:`Status`.
+Navigate to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
+:guilabel:`checkbox` in the top-left corner, beside the :guilabel:`Number` column, under the
+:guilabel:`New` button.
+
+This selects all existing vendor bills with a :guilabel:`Status` of :guilabel:`Posted` or
+:guilabel:`Draft`.
+
+Click the :icon:`fa-print` :guilabel:`Print` button to print the selected invoices or bills.
+
+Click :guilabel:`Register Payment` to create and process payments for multiple vendor bills at once.
 
-From here, click the :guilabel:`Action` gear icon to export, delete, or send & print the bills;
-click the :guilabel:`Print` icon to print the invoices or bills; or click :guilabel:`Register
-Payment` to create and process payments for multiple vendor bills at once.
+.. note::
+   Only payments with their :guilabel:`Status` listed as :guilabel:`Posted` can be billed in
+   batches. Payments in the :guilabel:`Draft` stage *must* be posted before they can be included in
+   a batch billing.
+
+Clicking :guilabel:`Register Payment` opens a :guilabel:`Register Payment` pop-up window. From the
+pop-up window, select the :guilabel:`Journal` the bills should post to, choose a :guilabel:`Payment
+Date`, and select a :guilabel:`Payment Method`.
 
-When :guilabel:`Register Payment` is selected, a pop-up appears. In this pop-up window, select the
-appropriate journal in the :guilabel:`Journal` field, choose a payment date in the
-:guilabel:`Payment Date` field, and choose a :guilabel:`Payment Method`. There is also the option to
-:guilabel:`Group Payments` on this pop-up, as well.
+There is also the option to :guilabel:`Group Payments` together from this pop-up window, as well. If
+this checkbox is ticked, only one payment is created, instead of one per bill.
```

Comment:
I'm actually not seeing this option in my run-through... is there a specific configuration or setting that needs to be enabled in order for this field to be visible? 
![Screenshot 2024-06-11 131651](https://github.com/odoo/documentation/assets/73958186/ee84616a-17b2-44e1-814f-d636be74b0a1)